VideoLocker/res/layout/fragment_my_all_videos.xml

```diff
@@ -8,16 +9,16 @@
     
     <!-- keep this container, as playerfragment needs it to be present -->
     <LinearLayout
-        android:id="@+id/container_player"
+        android:id="@+id/my_all_videos_player_container"
```

Comment:
What's the point of prefixing all the view IDs with `my_all_videos_`? The layout and Fragment already have that in their names.